src/entities/Votes/model.ts

```diff
@@ -27,18 +29,26 @@ export default class VotesModel extends Model<VoteAttributes> {
   }
 
   static async createEmpty(proposal_id: string) {
+    const newScore = this.newScoreFor(proposal_id)
+    await this.create(newScore)
+    return newScore
+  }
+
+  static async safelyGetVotesFor(proposal: ProposalAttributes<any>) {
+    let proposalVotes: VoteAttributes | null = await VotesModel.getVotes(proposal.id)
+    return proposalVotes === null || proposalVotes.votes === {} ? await VotesModel.createEmpty(proposal.id) : proposalVotes.votes
```

Comment:
the evaluation `proposalVotes.votes === {}` is always false, this method will create a new vote every time it is call